redis_enterprise/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
+services:
+  redis:
+    image: redislabs/redis:latest
```

Comment:
Could you pin this to a specific version? Otherwise the test might fail if the redis version changes.

Reply:
done

Reply:
sorry, where is the pin?

Reply:
https://github.com/DataDog/integrations-extras/commit/7d30649fe543942819e359b26e292632ff6951b6

i must have nuked it and not noticed

Reply:
mb github is having a bad day. I'll take another look tomorrow eu time.

Reply:
@j8-redis ultimately up to you. We have integrations where we essentially rely only on CI to run the tests (CI uses linux and windows). If you decide not to add these tests, please make sure to remove any "dead" code, i.e. this file and parts of `conftest.py` etc.